models/classes/class.QtiTestConverter.php

```diff
@@ -162,7 +162,11 @@ private function componentToArray(QtiComponent $component)
                         $array[$property->getName()][] = $item;
                     }
                 } else {
-                    $array[$property->getName()] = $value;
+                    if(is_string($value)) {   
+                        $array[$property->getName()] = html_entity_decode($value);
+                    }else{
+                        $array[$property->getName()] = $value;
+                    }
```

Comment:
Thanks, just a suggestion to simplify the code. We do not have ternary operators restrictions on BE side. 


```suggestion
                    $array[$property->getName()] = is_string($value) ? html_entity_decode($value) : $value;
```

Reply:
Thanks, indeed, looks better with ternary